packages/lisk-constants/tsconfig.browsertest.json

```diff
@@ -0,0 +1,13 @@
+{
+	"extends": "../../tsconfig",
+	"compilerOptions": {
+		"declaration": false,
+		"outDir": "browsertest.build",
+		"baseUrl": ".",
+		"rootDir": ".",
+		"paths": {
+			"chai": ["../../types/chai"]
+		}
+	},
+	"include": ["../../types/**/*", "src/**/*", "test/**/*"]
```

Comment:
I prefer to have this as a template file as well. Seems like we are gonna use this.